README.md

```diff
@@ -18,25 +18,27 @@ Yes! Papertrail provides a [free plan](https://papertrailapp.com/signup?plan=fre
 
 ## Logging format
 
-The library always reports the same system (`Particle`). And the application name is configurable. The log message itself follows the same pattern as the [SerialLogHandler](https://docs.particle.io/reference/firmware/photon/#logging) class.
+The library uses the Particle device ID for the Papertrail system. And the application name is configurable. The log message itself follows the same pattern as the [SerialLogHandler](https://docs.particle.io/reference/firmware/photon/#logging) class.
 
-For example a full log entry looks like this `Dec 23 00:09:09 Particle App:  [app] ERROR: An error occurred!`
+For example a full log entry looks like this `Dec 23 00:09:09 200000000555555555555555 App:  [app] ERROR: An error occurred!`
 
 ## Compatible Hardware:
 
-* Particle Photon is currently the only hardware tested to work with this library.
+* Particle Photon, Core and Electron were verified to work with this library.
+
+**Logging on the Electron consumes a lot of data. Be careful about how often you log.**
```

Comment:
Maybe `can consume a lot of data and may result in a higher bill` would be more accurate? It might also be nice to link the user to some docs on how to limit `how often you log`